src/esbmc/options.cpp

```diff
@@ -34,13 +34,15 @@ const struct group_opt_templ all_cmd_options[] = {
      "show SMT model (not supported by all solvers), if the formula is SAT"},
     {"color", NULL, "show colored output"}}},
   {"Trace",
-   {{"quiet",
-     NULL,
-     "do not print unwinding information during symbolic execution"},
-    {"compact-trace", NULL, ""},
-    {"symex-trace", NULL, "print instructions during symbolic execution"},
-    {"ssa-trace", NULL, "print SSA during SMT encoding"},
-    {"ssa-smt-trace", NULL, "print generated SMT during SMT encoding"},
+   {
+     {"quiet",
+      NULL,
+      "do not print unwinding information during symbolic execution"},
+     {"compact-trace", NULL, ""},
+     {"symex-trace", NULL, "print instructions during symbolic execution"},
+     {"ssa-trace", NULL, "print SSA during SMT encoding"},
+     {"ssa-smt-trace", NULL, "print generated SMT during SMT encoding"},
+     {"ssa-features-dump", NULL, "print features in the SSA (just before conversion)"},
     {"symex-ssa-trace", NULL, "print generated SSA during symbolic execution"},
     {"goto2c", NULL, "translate the GOTO program to C"},
     {"show-goto-value-sets",
```

Comment:
This is weird formatting. Not your PR. Just wondering whether clang-format is having a bad time with this options.cpp file and we should just do it in one go...